pandas/core/series.py

```diff
@@ -725,7 +725,7 @@ def __array_ufunc__(
             # it to handle *args.
             index = alignable[0].index
             for s in alignable[1:]:
-                index |= s.index
```

Comment:
we certainly are using this (or at least were), so need to make sure these inplace ops are deprecated as well.

Reply:
can you add some tests for the inplace to make sure they are warning as well